.env.example

```diff
@@ -175,6 +175,19 @@ GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
 GOOGLE_CALLBACK_URL=/oauth/google/callback
 
+# OpenID:
+# See OpenID provider to get the below values
+# Create random string for OPENID_SESSION_SECRET
+# For Azure AD
+#   ISSUER:  https://login.microsoftonline.com/(tenant id)/v2.0/
+#   SCOPE:   openid profile email
+OPENID_CLIENT_ID=
+OPENID_CLIENT_SECRET=
+OPENID_ISSUER=
+OPENID_SESSION_SECRET=
+OPENID_SCOPE=openid profile email
```

Comment:
I believe this will have to be wrapped in quotes as it contains spaces: `OPENID_SCOPE="openid profile email"`